components/dashboard/sections/Contributions.tsx

```diff
@@ -0,0 +1,355 @@
+      cell: ({ cell }) => {
+        const date = cell.getValue();
+        if (date) {
+          return <DateTime value={date} dateStyle="medium" timeStyle={undefined} />;
+        }
+      },
+    };
+
+    if (tab === 'recurring') {
+      return [
+        toAccount,
+        orderId,
+        paymentMethod,
+        amount,
+        totalDonations,
+        processedAt,
+        status,
+        {
+          header: 'Actions',
+          cell: ({ row }) => {
+            const order = row.original;
+            return (
+              <Flex justifyContent="center">
+                <PopupMenu
+                  placement="bottom-start"
+                  Button={({ onClick }) => (
+                    <StyledRoundButton data-cy="actions" size={32} onClick={onClick} buttonSize="small">
+                      <DotsHorizontalRounded size="24px" color={themeGet('colors.black.600')} />
+                    </StyledRoundButton>
+                  )}
+                >
+                  {() => (
+                    <Flex flexDirection="column">
+                      <ActionButton onClick={() => setEditOrder({ order, action: 'editPaymentMethod' })}>
+                        <FormattedMessage
+                          id="subscription.menu.editPaymentMethod"
+                          defaultMessage="Update payment method"
+                        />
+                      </ActionButton>
+                      <ActionButton onClick={() => setEditOrder({ order, action: 'editAmount' })}>
+                        <FormattedMessage id="subscription.menu.updateAmount" defaultMessage="Update amount" />
+                      </ActionButton>
+                      <ActionButton
+                        onClick={() => setEditOrder({ order, action: 'cancel' })}
+                        color={themeGet('colors.red.600')}
+                      >
+                        <FormattedMessage
+                          id="subscription.menu.cancelContribution"
+                          defaultMessage="Cancel contribution"
+                        />
+                      </ActionButton>
+                    </Flex>
+                  )}
+                </PopupMenu>
+              </Flex>
+            );
+          },
+        },
+      ];
+    } else {
+      return [toAccount, orderId, paymentMethod, amount, totalDonations, processedAt, status];
+    }
+  }
+};
+
+const TableWrapper = styled.div`
+  > div {
+    border-radius: 16px;
+  }
+
+  thead th {
+    padding: 16px 0px;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 18px;
+    color: ${props => props.theme.colors.black[700]};
+    :first-child {
+      padding-left: 20px;
+    }
+    :last-child {
+      padding-right: 20px;
+    }
+  }
+
+  tbody tr td {
+    font-size: 13px;
+    padding: 16px 0px;
+    color: ${props => props.theme.colors.black[800]};
+    :first-child {
+      padding-left: 20px;
+    }
+    :last-child {
+      padding-right: 20px;
+    }
+  }
+`;
```

Comment:
We should harmonize the table styles between the Agreements table and this contributions table (to look more like the contributions table) so we don't need to customize like this, but let's do that in a follow up.

Reply:
Yes. We definitely have a more consistent and styled data table component.